src/opera_disp_tms/generate_frame_tile.py

```diff
@@ -1,14 +1,14 @@
 import argparse
 import warnings
 from pathlib import Path
-from typing import Iterable, List
+from typing import Dict, Iterable, List
```

Comment:
Python 3.9 eliminated the need for Dict and List; you can use the builtin `dict` and `list` instead

`def foo(bar: list[str]) -> dict:`

Reply:
Done!